packages/browser-sdk/src/lib/react/VideoView.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useEffect, useRef } from "react";
+import * as React from "react";
```

Comment:
Why do we prefer this * import?

Reply:
React team has said that at one point they will stop supporting the default import `import React from "react";`. Not sure if/when it will happen, but I've just gotten into the habit of always doing `import * as React from "react"`. There's an article here with some more details: https://epicreact.dev/importing-react-through-the-ages/